content/english/structure/index.md

```
@@ -51,52 +51,68 @@ Core features of Steward Ownership, as implemented by Alkemio, include:
 
 The new organizational structure implemented by Alkemio is shown below. All named entities are registered in The Netherlands.
 
-![Alkemio Group Structure](/images/alkemio-group-structure.svg)
+![Alkemio Group Structure](/legal/alkemio-group-structure-2025.svg)
 
 **Alkemio Holding** is the Steward Ownership entity. It is a commercially and professionally run business to achieve the purpose. The value generated is distributed in accordance with the principles of Steward Ownership and the purpose.
 
+**Stichting Alkemio** is the operational governance for Alkemio Holding. The board of Stichting Alkemio are the stewards.
+
+**Stichting Golden Share** is an independent foundation whose purpose is to provide a veto service for steward-owned companies. It is the holder of the Purpose Control (PC) share. There are provisions in the legal documents that state that approval is required from the PC holder. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)